### DISCUSSION TITLE
防駭客的基本認知：資料保護

### DISCUSSION THREAD

**Discussion by Rosancl** [1101-GS4719-C/Python-Community-GS4719-C#418] [General]
現在越來越多人體認到資安的重要性，這篇文章簡單介紹了一些個人資料可能外流的原因和方式，也提供了一些在保護個資上的方式供參考。 防範的方式有設立帳密或信用卡多重驗證、瀏覽器安全設定等等，其中提到的10分鐘信箱是我以前不知道的方式，沒想到還可以在申請一些網站帳號的時候使用免洗信箱，以保護自己的信箱。 分享給大家 👍 

https://www.bnext.com.tw/article/67961/data-leakage

**Comment by Lai27330** [comment]
雖然說多重驗證的確能增強保護力，但是如果記不住那麼多密碼怎麼辦而且對多數人來說簡單的操作方式仍然是優先考量

**Comment by pekoranoinu** [comment]
感謝樓主分享的防駭客秘訣

**Comment by chupochun** [comment]
謝謝分享保護信箱的方法，這樣就不用擔心自己的信箱會被其他人看光了

**Comment by JSHT** [comment]
一講到 temp mail，就讓我想到現在 slader 搬到 quizlet 要有 plus 帳號。就可以用 temp mail 刷 30 天的使用權限。

**Comment by lx3211** [comment]
10分鐘信箱的確是對我來說很新奇的方式
但是如果用的是十分鐘信箱去註冊帳號
以後忘記密碼就找不回來了( ･᷄ὢ･᷅ )

**Comment by lx3211** [comment]
我會比較頃向用假資料申請一個新的信箱
然後比較不重要、繁瑣的網站就用這個信箱去註冊

**Comment by Ku-Hao-Chen** [comment]
感謝分享，讓我對資料保護有基本的認識。

**Comment by Lynn12378** [comment]
資料保護真的很重要，之前一個高中同學在網路上買東西，交易資料被外洩，結果就被詐騙8千多。其他像是平常接到補習班電話，似乎也是資料外洩的影響，甚至我在去年剛考完指考，就收到儒林的宣傳單，真的非常可怕，感謝樓主分享怎麼保護個資。

**Comment by xkllkx** [comment]
謝謝分享，信箱真的很重要，尤其現在許多程式、app都與google帳戶綁定，如果被盜，後果不堪設想。

**Comment by yucheng1028** [comment]
感謝分享，十分鐘信箱對我來說很有用，可以免除收廣告信及個資安全上的困擾。

**Comment by ghost** [comment]
個資現在真的都快不值錢了..
各大企業也正因為有了這些才能使的演算法越趨齊全

**Comment by bloodtrail0511** [comment]
除了10分鐘信箱之外，使用vpn也是不錯的方法。可以保護線上隱私，讓ip位置和線上活動不外流

**Comment by leon10456** [comment]
感謝分享 已馬上更改瀏覽器的安全性設定！

**Comment by sean1325418** [comment]
陌生的網站最好還是不要亂點，不然資料怎麼被偷走的都不知道

**Comment by PoooYen** [comment]
感謝分享！陌生的網站一律無痕加訪客登入

**Comment by 11timyyy** [comment]
推無痕真的超棒!大家也要小心自己的個資不要隨便在陌生裝置上登入

**Comment by HuanYuuuuuu** [comment]
感謝分享! 個人資料的保護真的非常重要 不然等到資料被盜竊後就來不及了。
